### PR TITLE
Modify header

### PR DESCRIPTION
**Pull request by jerrylee17**
- Fix border radius
- Reduce header margin

For pages that do not by default have padding on the top -- I'm adding padding to it

### CODE REVIEW

**Comment by qsaml** [comment]
> Screen Shot 2020-08-28 at 11 38 55 PM get rid of the orange overflow

new coder question: was that always there? and how did it get there in the first place?

**Comment by IsitaB** [comment]
> > Screen Shot 2020-08-28 at 11 38 55 PM get rid of the orange overflow
> 
> new coder question: was that always there? and how did it get there in the first place?

It used to be taller

File: src/Components/Header/Header.css [hunk 1]
```diff
@@ -1,13 +1,15 @@
 .header-jumbo {
-  padding: 7rem 2rem;
+  padding: 2rem 2rem;
   background-image: url("./assets/background1.jpg");
-  background-size: 100%
+  background-size: 100%;
+  border-radius: 0px !important;
+  margin-bottom: 10px !important;
```
Comment (IsitaB) [inline src/Components/Header/Header.css:6]: this

File: src/Pages/EventManager/EventManager.js [hunk 2]
```diff
@@ -50,9 +50,9 @@ export default function EventManager(props) {
   async function handleSubmit(event) {
     if (modalState === eventModalState.SUBMIT) {
       const res = await createNewEvent(event, props.user.token);
-      if(res.error === false) {
+      if (res.error === false) {
```
Comment (IsitaB) [inline src/Pages/EventManager/EventManager.js:53]: is this the right way? why'd you touch this?

File: src/Pages/PrintingAnalytics/PrintingAnalytics.js [hunk 3]
```diff
@@ -143,13 +150,14 @@ export default function Analytics() {
...
           dropdownMode='select'
+          id='date-select'
         />
       </div>
       <div className='visualizations'>
         <div className='line-graph'>
           <Line
             data={lineGraphInfo}
-            options={{responsive: true, maintainAspectRatio: true}}
+            options={{ responsive: true, maintainAspectRatio: true }}
```
Comment (IsitaB) [inline src/Pages/PrintingAnalytics/PrintingAnalytics.js:160]: is this the right way?

File: src/Pages/EventManager/EventManager.js (hunk 2 above)
Comment (jerrylee17) [reply src/Pages/EventManager/EventManager.js:53]: autoformat

File: src/Pages/PrintingAnalytics/PrintingAnalytics.js (hunk 3 above)
Comment (jerrylee17) [reply src/Pages/PrintingAnalytics/PrintingAnalytics.js:160]: also autoformat

**Pull request merged**